src/quo/components/text_combinations/view.cljs

```diff
@@ -28,15 +42,20 @@
            button-icon
            button-on-press
            customization-color
-           emoji-hash]}]
+           emoji-hash
+           emoji
+           emoji-background-color]}]
   [rn/view {:style container-style}
    [rn/view
     {:style {:flex-direction  :row
              :justify-content :space-between}}
     [rn/view {:style style/title-container}
      (when avatar
        [rn/view {:style style/avatar-container}
-        [icon avatar 32]])
+        [icon avatar 32 nil nil]])
```

Comment:
You can just make `icon` take a single map and then you won't need to pass nils.